docs/application/vscode-ext/web.md

```diff
@@ -333,46 +333,106 @@ To debug application, follow these steps:
+
+For more information on each function, see:
+- [Tizen Web: Build Tizen App](#Build-Project)
+- [Tizen Web: Debug Tizen App](#Deploy-and-Run-Application-in-Emulator)
+- [Tizen Web: Install Tizen App](#Deploy-and-Run-Application-in-Emulator)
+- [Tizen Web: Make Exclude Build List](#Exclude-Build-List)
+- [Tizen Web: Run Tizen App](#Deploy-and-Run-Application-in-Emulator)
+
+![Design Editor launch](media/tizen_web34.png)
+
 
-To run Design Editor, follow one of the below ways:
+## Tizen Web with Design Editor
 
-> **Note** 
-> 
-> You must select the **HTML extension file** in the file explorer.
+Visual Studio Code Extension for Tizen Web supports WYSIWYG Design Editor features such as **Preview**, **JavaScript Assistant**, **Structure View**, **New Page**, **HTML Assistant** , **Undo** , and **Redo**.
+You must select the **HTML extension file** in the file explorer before you launch Design Editor.
 
-- Command Palette
-    1. In VS Code, click **View** > **Command Palette**.
-    2. In the input field of the **Command Palette** that appears, type **Tizen Web: Run Design-Editor** and press **Enter**.
+### Run Design Editor
+
+To run Design Editor, perform one of the following:
 
-        A new tab is created and the Design Editor is launched.
+- Using Command Palette
+    1. To launch Design Editor, in VS Code, click **View** > **Command Palette**.
+    2. In the input field of **Command Palette** that appears, type **Tizen Web: Run Design-Editor** and press **Enter**.
 
-- Design-Editor button
+        A new tab is created and Design Editor is launched.
 
-    To launch the Design Editor, click **Design-Editor** at the bottom right of the screen.
+- Using Design-Editor button
+
+    To launch Design Editor, click **Design-Editor** on the VS Code window as shown in the following figure:
 
     ![Design Editor button](media/tizen_web27.png)
-- Right-click preference
-    1. Select the HTML file that you want to launch and right-click.
+
+- Using right-click options
+    1. To launch Design Editor, select the HTML file that you want to launch and right-click.
     2. From the list that appears, click **Tizen Web: Run Design-Editor**.
 
         ![Design Editor right click](media/tizen_web28.png)
 
-Design Editor is successfully launched.
+### Features
 
-![Design Editor launch](media/tizen_web29.png)
+Following are the WYSIWYG Design Editor features:
 
-### Quick Action Execution
-Right-click on the **project folder** to use the extension function in the menu.
+- Drag and Drop a component
 
-For more information on each function, see:
-- [Tizen Web: Build Tizen App](#Build-Project)
-- [Tizen Web: Debug Tizen App](#Deploy-and-Run-Application-in-Emulator)
-- [Tizen Web: Install Tizen App](#Deploy-and-Run-Application-in-Emulator)
-- [Tizen Web: Make Exclude Build List](#Exclude-Build-List)
-- [Tizen Web: Run Tizen App](#Deploy-and-Run-Application-in-Emulator)
+    You can drag and drop a component from the Widget palette to the canvas.
```

Comment:
@singa2000 In the image, the UI marked by red box is titled as "Widgets".
If so, isn't "Widgets palette" correct as the name?

Reply:
Widgets palette is correct.  I changed